cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -139,9 +147,11 @@ write_files:
         vendor_cookbook
       fi
       cd /tmp
+      mkdir -p /etc/chef/ohai/hints
+      touch /etc/chef/ohai/hints/ec2.json
 
       start=$(date +%s)
-
+      jq -s ".[0] * .[1]" /tmp/dna.json /tmp/extra.json > /etc/chef/dna.json || ( echo "jq not installed"; cp /tmp/dna.json /etc/chef/dna.json )
```

Comment:
The error message may be misleading. Here we are assuming that the command would fail only if jq is not installed, but it could fail for other reasons. I know this is something coming from the change we are reverting, so not worth it to fix in this PR, but I suggest to improve in at least in a follow up.

Reply:
I am actually planning on removing the message in next iteration. As we install `jq` as part of cookbook recipies. Meanwhile since we are using a native approach we do not need to worry about these files being availble or not as this is the create Path and these files are created using write_files directives,  if they are not created then we never reach this stage at all!

But I will make change in the `cfn-hup-update-action.sh` that we add in Update Path. Thanks!!